pkg/policycache/cache_test.go

```diff
@@ -635,3 +728,37 @@ func Test_Ns_Add_Remove_User(t *testing.T) {
 		t.Errorf("expected 0 validate enforce policy, found %v", len(deletedValidateEnforce))
 	}
 }
+
+func Test_Mutate_Policy(t *testing.T) {
+	pCache := newPolicyCache(log.Log, dummyLister{}, dummyNsLister{})
+	policy := newMutatePolicy(t)
+	//add
+	pCache.Add(policy)
```

Comment:
Let's "add" and "get" the same policy multiple times.

And please cover namespace policy as well. we can use the same policy name and try to get both policies from the cache.